icechunk-python/src/lib.rs

```diff
@@ -84,6 +103,7 @@ fn _icechunk_python(py: Python<'_>, m: &Bound<'_, PyModule>) -> PyResult<()> {
     m.add_class::<PyDiff>()?;
     m.add_function(wrap_pyfunction!(initialize_logs, m)?)?;
     m.add_function(wrap_pyfunction!(spec_version, m)?)?;
+    m.add_function(wrap_pyfunction!(cli_entrypoint, m)?)?;
```

Comment:
I think you will want to also add the new function in the __init__ file and in the .pyi file. Youll see all other function in there too.

Reply:
So it seems necessary to add the function here to install it through `[project.scripts]`, but I'm not sure init/pyi is needed – this should never be called from Python. What do you think?